src/components/feedback/index.tsx

```diff
@@ -0,0 +1,98 @@
+"use client";
+
+import InfoSvg from "@/assets/svgs/InfoSvg";
+import CommonInput from "@/components/common/inputs/input";
+import CommonInputFile from "@/components/common/inputs/fileInput";
+import CommonLabel from "../common/containers/label";
+import React from "react";
+import { Boat } from "@/models/models";
+import { useTranslation } from "react-i18next";
+import StarRatings from "react-star-ratings";
+import SubmitButton from "../common/containers/submit-button";
+import { useEffect, useState } from "react";
+
+const FormWrapper = ({ children }: { children: React.ReactNode }) => {
+  return <div className="relative w-[48%]">{children}</div>;
+};
+
+export default function FeedbackForm({
+  data,
+  setData,
+  boatInfo,
+}: {
+  data: any;
+  setData: any;
```

Comment:
We keep with the any here. 

data => Booking
setData => I don't think is needed